src/tests/Common/helixpublishwitharcade.proj

```diff
@@ -236,7 +236,12 @@
     <MSBuild Projects="@(_ProjectsToBuild)" Targets="CreateTestEnvFiles" StopOnFirstFailure="true" />
   </Target>
 
-  <Target Name="ZipPayloadDirectory" AfterTargets="PreparePayloadsDirectories" Condition="'$(TargetOS)' == 'Android'">
+  <PropertyGroup>
```

Comment:
FYI - the location in the file shouldn't matter, the condition is evaluated at the time of running the target afaik

Reply:
Thanks for the info. I moved it because It is also easier for the reader of this file to know what that var is.

Reply:
Got it